### PR TITLE
Add support for markers and detached audio to FCP X adapter

### PR DESCRIPTION
**Pull request by eric-with-a-c**
This adds support for markers and audio that's been detached from the video clip.

I also updated the documentation a little too, but that may not be desired so let me know and I'll revert those changes.

### CODE REVIEW

File: docs/tutorials/feature-matrix.rst [hunk 1]
```diff
@@ -4,26 +4,26 @@ Feature Matrix
...
-+--------------------------+----------+----------------+---------+-----------+------------+-----------+
-|Audio/Video Effects       | planned  | needs research | planned | planned   | planned    | no        |
-+--------------------------+----------+----------------+---------+-----------+------------+-----------+
-|Speed Effects             | yes      | yes            | planned | read-only | planned    | no        |
-+--------------------------+----------+----------------+---------+-----------+------------+-----------+
-|Color Decision List (CDL) | metadata | yes            | planned | planned   | planned    | planned   |
-+--------------------------+----------+----------------+---------+-----------+------------+-----------+
\ No newline at end of file
++--------------------------+----------+----------------+---------+-----------+------------+-----------+-----------+
+|Feature                   | OTIO     | EDL            | FCPXML  | AAF       | RV         | ALE       | FCPX XML  |
```
Comment (ssteinbach) [inline docs/tutorials/feature-matrix.rst]: After talking with @jminor, we feel like putting a 7 in FCP7 XML to clarify FCP7 vs FCPX will help.  Thanks again!

```suggestion
|Feature                   | OTIO     | EDL            | FCP7 XML  | AAF       | RV         | ALE       | FCPX XML  |
```

Comment (eric-with-a-c) [reply docs/tutorials/feature-matrix.rst]: Sounds like a good idea. I made the change.

File: docs/tutorials/adapters.md [hunk 2]
```diff
@@ -10,7 +10,7 @@ Final Cut 7 XML Format
 - <a href="https://developer.apple.com/library/content/documentation/AppleApplications/Reference/FinalCutPro_XML/AboutThisDoc/AboutThisDoc.html#//apple_ref/doc/uid/TP30001152-TPXREF101" target="_blank">Reference</a>
 
 Final Cut Pro X XML Format:
-- <a href="https://github.com/PixarAnimationStudios/OpenTimelineIO/issues/37">Status</a>
+- Status: Supported via the `fcp_xml` adapter
```
Comment (ssteinbach) [inline docs/tutorials/adapters.md]: Sorry, one more thing, this should be fcpx_xml, not fcp_xml, right?

```suggestion
- Status: Supported via the `fcpx_xml` adapter
```

Comment (eric-with-a-c) [reply docs/tutorials/adapters.md]: Yup, it should be fcpx_xml

**Comment by eric-with-a-c** [comment]
Let me know if you find anything else that needs to be fixed 😄

**Pull request merged**